streaming-payments/Anchor.toml

```diff
@@ -2,14 +2,14 @@
 seeds = false
 skip-lint = false
 [programs.localnet]
-psp_payment_streaming = "Fg6PaFpoGXkYsidMpWTK6W2BeZ7FEfcYkg476zPFsLnS"
+psp_streaming_payments = "Fg6PaFpoGXkYsidMpWTK6W2BeZ7FEfcYkg476zPFsLnS"
 
 [registry]
 url = "https://api.apr.dev"
 
 [provider]
 cluster = "Localnet"
-wallet = "/home/ananas/.config/solana/id.json"
+wallet = "/Users/tsv/.config/solana/id.json"
```

Comment:
would this work?
```suggestion
wallet = "$HOME/.config/solana/id.json"
```

Reply:
I think `~/.config/solana/id.json` should work